### ISSUE TITLE
U.S. Fed Gov  Web Design Standards - Investigate use for VDP

### ISSUE THREAD

**Issue by rafael5** [vistadataproject/documents#19]
###  US government web design standards (background)
* http://www.digitalgov.gov/resources/checklist-of-requirements-for-federal-digital-services/
* http://www.theatlantic.com/technology/archive/2015/10/the-us-governments-new-design-standards/410986/

### U.S. Digital Services  Web Style Guide:
https://18f.gsa.gov/2015/09/28/web-design-standards
https://playbook.cio.gov/designstandards
https://playbook.cio.gov/designstandards/getting-started

### Other government Web Design Style Guides:
* https://styleguide.healthcare.gov
* http://uspto.github.io/designpatterns/
* https://cfpb.github.io/design-manual/

### HHS?
*  HHS has generated hundreds of pages of legal 'standards' and requirements, 
* Its website contains not a single example, screenshot, nor any implementation guidance:  http://webstandards.hhs.gov/standards
* HHS's Github?  Only a placeholder.  No code.   https://github.com/HHS/HHS-Responsive-Design
* In summary:  Tens of thousands of lines of policy, regulation, and requirements.  Zero lines of implementable code.  No advice nor technical means to implement or satisfy any of these requirements.
